src/transformStateWithClones.js

```diff
@@ -7,7 +7,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  let cloneState = { ...state };
```

Comment:
check this checkpoint
![image](https://user-images.githubusercontent.com/122989585/215569404-d98ca22e-aa53-4920-9c5c-49184ecacfd6.png)